components/GetInvolved.tsx

```diff
@@ -108,70 +108,56 @@ export default function GetInvolved() {
 						<div id="body-left-content" className="w-5/12 max-w-sm">
 							<h2
 								id="body-left-header"
-								className=" font-dmserif text-4xl xl:text-5xl">
+								className="font-dmserif font-light tracking-wide text-4xl xl:text-4xl pr-12">
 								Join us in making a difference!
 							</h2>
 							<p
 								id="body-left-text"
-								className="montserrat mt-8 text-sm font-medium leading-6 lg:text-base">
-								Donate to SPEC today to <br></br> support our mission and
+								className="font-montserrat mt-8 text-sm font-medium leading-6 lg:text-base">
+								Donate to SPEC today to support <br></br> our mission and
 								impact.
 							</p>
 							<div
 								id="container-for-anchor-button"
-								className="mt-5 flex max-w-[15rem]  items-center justify-between">
-								<a
-									id="learn-more-hyperlink"
-									className="montserrat font-semibold tracking-wide underline"
-									href="javascript:void(0)"
-									onClick={() => setDonationTable("table")}>
-									<p className="whitespace-nowrap text-sm">LEARN MORE</p>
-								</a>
+								className="mt-5 flex max-w-[15rem] items-center justify-between">
+								<p>
+									<a
+										id="learn-more-hyperlink"
+										className="font-montserrat text-sm mr-24 text-gray-400 font-semibold tracking-wide underline"
+										href="javascript:void(0)"
+										onClick={() => {
+											const url = "https://docs.specollective.org/introduction/";
+											window.open(url, '_blank');
+										}}>
+										LEARN MORE
+										</a>
+								</p>
 								<button
 									id="donate-button"
 									type="button"
-									className="montserrat max-w-28 mx-2 w-24 rounded-br-3xl rounded-tl-3xl bg-spec-turquiose px-5 py-3 text-center text-xs font-semibold tracking-wider text-white"
+									className="font-montserrat w-40 h-12 rounded-br-3xl rounded-tl-3xl bg-spec-turquiose text-center text-xs font-semibold tracking-wider text-white"
 									onClick={() => {
-										location.href = "https://opencollective.com/spec";
+										const url = "https://opencollective.com/spec";
+    								window.open(url, '_blank');
 									}}>
 									DONATE
 								</button>
 							</div>
 						</div>
-						{donationTable === "Image" ? (
-							<div id="body-right-content" className="scale-90">
-								<Image
-									id="donate-image"
-									className=""
-									src="/DonateImg.svg"
-									alt="donate-image"
-									width={500}
-									height={500}
-								/>
-							</div>
-						) : (
-							<div id="donation-table">
-								<div className="flex justify-end" data-closeable>
-									<button
-										type="button"
-										onClick={() => setDonationTable("Image")}>
-										<span aria-hidden="true" className="">
-											&times;
-										</span>
-									</button>
-                  </div>
+						<div id="donation-table">
```

Comment:
Here you want to add a className="lg:w-1/2" to the div with the id "donation-table"
In the div directly underneath (that wraps the donation table component" add this className="flex justify-center lg:w-full"
You'll still have to adjust the left margins on the left half of get involved section but will get you closer to the figma